tooling/minimize/tests/ub/tree_borrows/core/cell_inside_slice.stderr

```diff
@@ -1 +1 @@
-fatal error: UB: Tree Borrows: writing to the local of a pointer with Frozen permission
+fatal error: UB: Tree Borrows: Read-only violation: local write to a read-only reference (shared, or mutable after a foreign write)
```

Comment:
There are a lot of colons in this error message. Is this "Read-only violation:" prefix really useful?

I would probably say:
"local write to a read-only reference (shared reference, or mutable reference that previously encountered a foreign write)"